configs/inspection/Square.xml

```diff
@@ -3,5 +3,6 @@
   <option name="myName" value="Square" />
   <inspection_tool class="FallthruInSwitchStatement" enabled="true" level="ERROR" enabled_by_default="true" />
   <inspection_tool class="UnnecessarySemicolon" enabled="true" level="ERROR" enabled_by_default="true" />
+  <inspection_tool class="MoveLambdaOutsideParentheses" enabled="false" level="WEAK WARNING" enabled_by_default="false" />
```

Comment:
Does disabling this remove the quick fix? I noticed you can change the level to "No highlighting, only fix" which keeps the quick fix but doesn't warn you.

Reply:
Not sure what the XML syntax is for that.

Reply:
Hmm, good point.  I'll test.